tests/karate/src/test/java/common/net/MultiMsgWebSocketClient.java

```diff
@@ -37,4 +41,27 @@ public synchronized Object listen(long timeout) {
   public MessageBuffer getBuffer() {
     return queue;
   }
+
+  public void publish(String data, int id, String channel){
+    JsonConverter jsonConverter = new JsonConverter();
+    Json request =  jsonConverter.publishМessageFromData(data, id, channel);
+    this.send(request.toString());
+  }
+
+  public String getBackendResponseWithBroadcast(String a){
+    String broadcast = getBuffer().takeTimeout(5000);
+    String result = getBuffer().takeTimeout(5000);
```

Comment:
how do you know which one is the broadcast and which one is the result ?
I don't think the order was specified in the protocol, and it's probably fine to leave it undefined.